From 580c3256e181a18b9bdb40bfd109c2c43ba9afb3 Mon Sep 17 00:00:00 2001
From: Paul Gevers <elbrus@debian.org>
Date: Sun, 28 Jul 2024 17:14:03 +0200
Subject: [PATCH 1/2] tests: don't fail on output to stderr and mark it
 superficial

The test currently doesn't work on ci.debian.net infrastructure because
$(hciconfig) doesn't seem to find any device and prints
"""
Can't open HCI socket.: Address family not supported by protocol
"""

to stderr. Autopkgtest by default fails when a test writes to stderr, so let's
not fail the test on that. However, as all tests are now skipped, in the
current for it can only be superficial, i.e. a passing test has no guarantees
about the functioning of the package.

Closes: #1071147
---
 debian/tests/control | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/tests/control b/debian/tests/control
index c710150..4168957 100644
--- a/debian/tests/control
+++ b/debian/tests/control
@@ -1,3 +1,3 @@
 Tests: bluez-response
 Depends: python3-dbus:native, bluez, bluez-test-scripts
-Restrictions: needs-root, isolation-container
+Restrictions: needs-root, isolation-container, allow-stderr, superficial
-- 
2.43.0

